### ISSUE TITLE
Cannot set custom `userData`

### ISSUE THREAD

**Issue by asm0dey** [vendelieu/telegram-bot#103]
In [docs](https://github.com/vendelieu/telegram-bot/wiki/Bot-Context) there is a way to set custom implementation of `userData`, but `userData` is internal and this can not be modified.

**Comment by vendelieu** [comment]
Thanks for enlightening a non-obvious point, I'll fix it in the documentation to make it clearer :)

In order to logically separate heaps of parameters into different groups in the config, dsl functions are used, for the current task try wrapping in the `context{}` function:
```
TelegramBot("...") {
    context {
        userData = MyImpl()
   }
}
```

**Comment by vendelieu** [comment]
Thanks for the report!

**Comment by vendelieu** [comment]
fixed in v5.0.1
